src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -78,7 +82,7 @@ export const StyledContentLink = styled.a`
   }
 
   ${theme.breakpoints.mobile(css`
-    margin: 1.2rem;
+    padding: 1.2rem;
```

Comment:
Padding makes the clickable area of `StyledContentLink` much bigger. I'm not sure if it's wrong, but I think you should keep that in mind.

Reply:
that's okay